std_msgs/Header header
uint8 rows
uint8 cols
float32 cell_width
float32 cell_height
uint16 full_scale_output
uint16[] data